tests/trading/test_completeSets.py

```diff
@@ -68,7 +68,7 @@ def test_publicBuyCompleteSets_failure(contractsFixture):
 
     # buyCompleteSets exceptions
     with raises(TransactionFailed):
-        completeSets.publicBuyCompleteSets(market.address + 1, fxpAmount, sender=tester.k1)
+        completeSets.publicBuyCompleteSets(0, fxpAmount, sender=tester.k1)
```

Comment:
market.address is now a hex string so the operation failed. I can modify this to be more like it was.